src/frontend/screens/Game/GamePage/index.css

```diff
@@ -276,6 +275,7 @@
         & .mainBtn {
           min-width: 200px;
           font-size: var(--text-md);
+          white-space: nowrap;
```

Comment:
button test going to 2 lines was overflowing

![image](https://github.com/user-attachments/assets/adaa591f-09dc-4f3a-ab03-82b7de874561)

and now

![image](https://github.com/user-attachments/assets/543c9003-4904-404a-b370-f0816f671eec)